ldms/src/ldmsd/ldmsd.c

```diff
@@ -2241,13 +2245,12 @@ int main(int argc, char *argv[])
 	ldms_version.flags, OVIS_GIT_LONG
 
 #if OVIS_LDMS_HAVE_AUTH
-				fprintf(bfile, BANNER_PART1_A
+			fprintf(bfile, BANNER_PART1_A
```

Comment:
@nichamon, @valleydlr did we talk about getting rid of this banner file?

Reply:
According to the spreadsheet we went over together in Austin (I left early), it was decided to remove the command-line option `-B` and add a new config command to replace the option.

Reply:
@valleydlr, do you want to retain the Banner file?